### DISCUSSION TITLE
Code lines doubled

### DISCUSSION THREAD

**Discussion by tylerwilsontrustage** [danielpalme/ReportGenerator#694] [Q&A]
Having an issue running Report Generator where the final report has code lines doubled and coverage halved. This is a big problem for code quality gates of course. This started after upgrading all the projects to .NET 8.0. Getting a bunch of these messages in the log.

2024-09-13T17:31:45: Error during reading file '[https://dev.azure.com/redacted'](redacted%27): Response status code does not indicate success: 500 (Internal Server Error).

Yeah, there's an extra %27 at the end to, weird. That's a single-quote character.

The Code Coverage Report artifact also contains only one summary folder where it used to contain two. No idea if that's relevant.

**Comment by danielpalme** [comment]
Can you share the coverage files you are using? I don't need your source code.

You can send them privately via email: [reportgenerator@palmmedia.de](mailto:reportgenerator@palmmedia.de) or share them here.

Please let me know one class where I can see the issue.

**Comment by danielpalme** [comment]
Thanks for the files!
I created the HTML report based on the 16 cobertura files you have sent.
Can you point me to one class where code lines are doubled? Maybe a screenshot can describe your problem.
